aiosmtpd/tests/test_main.py

```diff
@@ -31,7 +31,7 @@ def tearDownModule():
     ModuleResources.close()
 
 
-class TestHandler1:
+class HandlerWithFromCli:
```

Comment:
Classes starting with `Test` get picked up by pytest as Test Classes, causing failures.

I've renamed these classes to something that better reflect what they'll be doing.